cfspopcon/formulas/__init__.py

```diff
@@ -24,9 +24,24 @@
 from .divertor_metrics import calc_B_pol_omp, calc_B_tor_omp
 from .energy_confinement_time_scalings import calc_tau_e_and_P_in_from_scaling
 from .figures_of_merit import calc_normalised_collisionality, calc_peak_pressure, calc_rho_star, calc_triple_product
+from .fluxes import calc_breakdown_flux_consumption, calc_flux_external, calc_flux_internal, calc_flux_PF, calc_flux_res
 from .fusion_rates import calc_fusion_power, calc_neutron_flux_to_walls
-from .geometry import calc_plasma_surface_area, calc_plasma_volume
+from .geometry import calc_plasma_poloidal_circumference, calc_plasma_surface_area, calc_plasma_volume
 from .impurity_effects import calc_change_in_dilution, calc_change_in_zeff, calc_impurity_charge_state
+from .inductances import (
+    calc_external_inductance,
+    calc_fa,
+    calc_fb,
+    calc_fc,
+    calc_fd,
+    calc_fg,
+    calc_fh,
```

Comment:
Are these functions used standalone or just internally to the other inductance functions?

Reply:
These are only necessary insofar as there as used in tests